src/view/ThemeManager.js

```diff
@@ -321,6 +322,44 @@ define(function (require, exports, module) {
         return loadFile(fileName, themePackage.metadata);
     }
 
+    /**
+     * Loads a remote theme by stubbing the File object expected by 
+     * the Theme constructor. It must contain: 
+     *
+     * {
+     *   name: {String} filename of the theme's LESS file
+     *   _path: {String} URL path to the theme's LESS file
+     * }
+     *
+     * Likewise, the options object must contain whatever metadata
+     * is found in the theme's package.json file:
+     *
+     * {
+     *   theme: {
+     *     // Metadata here.
+     *   }
+     * }
+     *
+     * @param {Object} pre-populated file object.
+     * @param {Object} metadata options for the theme
+     * @return {Theme} the Theme object created for this theme
+     */
+     function manualAddTheme(file, options) {
```

Comment:
Just `addTheme()` is fine.  Everything you call in code is done so "manually", so it's not necessary.